tests/functional_tests/io-tests.bats

```diff
@@ -3,41 +3,55 @@
 # all the tests related to HackerRank compatible IO tests for DSA course
 # setup and teardown functions
 setup() {
-	echo "empty setup"
+	sed -i 's/unit_tests/io_tests/' ../../execution_nodes/extract_run.sh
+	mkdir $BATS_TMPDIR/io-tests-example
 }
 
 teardown() {
-	echo "empty teardown"
+    sed -i 's/io_tests/unit_tests/' ../../execution_nodes/extract_run.sh
+	rm -rf $BATS_TMPDIR/io-tests-example
```

Comment:
@kashyapgajera  I don't understand you statement:
"Too many code climate issues will pop up if I make small changes that were suggested." Please be more specific and points to lines of code with appropriate comment.